### ISSUE TITLE
Issue with parked domain

### ISSUE THREAD

**Issue by nhanledev** [QROkes/webinoly#15]
**Issue**
1. Error when creating parked domain
2. It seems like the config include part: `include /var/www/testdomain.com/*-nginx.conf;` is incorrect. It should be `include /var/www/maindomain.com/*-nginx.conf;` because it is the parked domain and need to load all the configuration of the main one.

**Step to reproduce:**
Assume that maindomain.com and testdomain.com are all registered and correctly DNS'ed.

1. Create main domain:
`sudo site maindomain.com -php`
`sudo site maindomain.com -force-redirect=root`
`sudo site maindomain.com -ssl-on`

2. `sudo site -testdomain.com -parked=maindomain.com`

Bug on screen:

```text
Site testdomain.com has been successfully created!

Main site domain: maindomain.com


Parked domain was successfully configured! 

Job for nginx.service failed because the control process exited with error code.
See "systemctl status nginx.service" and "journalctl -xe" for details.
```

**More information:**
`sudo nginx -t` error:
```text
nginx: [warn] conflicting server name "testdomain.com" on 0.0.0.0:80, ignored
nginx: [warn] conflicting server name "www.testdomain.com" on 0.0.0.0:80, ignored
nginx: [warn] conflicting server name "testdomain.com" on [::]:80, ignored
nginx: [warn] conflicting server name "www.testdomain.com" on [::]:80, ignored
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
```

Content of `testdomain.com`
```text
# WebinolyWWWredirectStart - Force WWW or non-WWW redirect
server {
	listen 80;
	listen [::]:80;
		server_name testdomain.com www.testdomain.com;

	include common/headers-http.conf;
	return 301 $scheme://testdomain.com$request_uri;
}
# WebinolyWWWredirectEnd
# WebinolyNginxServerStart
server {
	listen 80;
	listen [::]:80;

		server_name testdomain.com www.testdomain.com;

	access_log /var/log/nginx/testdomain.com.access.log we_log; 
	error_log /var/log/nginx/testdomain.com.error.log;

	root /var/www/maindomain.com/htdocs;
		
	index  index.php index.html index.htm;
		
	include common/php.conf;
	include common/locations.conf;
	include common/headers-http.conf;
	include common/headers-html.conf;
	include /var/www/testdomain.com/*-nginx.conf;
}
# WebinolyNginxServerEnd

```

**What the configuration file should be**
I have to manually fix the config file to this to solve the problem, and to make the `sudo site testdomain.com -ssl-on -root=maindomain.com` works.

```text
# WebinolyNginxServerStart
server {
        listen 80;
        listen [::]:80;

        server_name testdomain.com;

        access_log /var/log/nginx/testdomain.com.access.log we_log;
        error_log /var/log/nginx/testdomain.com.error.log;

        root /var/www/maindomain.com/htdocs;

        index  index.php index.html index.htm;

        include common/php.conf;
        include common/locations.conf;
        include common/headers-http.conf;
        include common/headers-html.conf;
        include /var/www/maindomain.com/*-nginx.conf;
}
# WebinolyNginxServerEnd
```

**Comment by QROkes** [comment]
Thanks @nhanledev 
This issue was unintentionally introduced in v1.5.0 and will be fixed in the next minor release.

**Comment by QROkes** [comment]
Fixed in v1.6.1.
